integration/stability/bind_mount_linux.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Ning Lu
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will start a container with a bind mount
+# and set bind propagation, the purpose of this
+# test is to check if the container stops properly
+
+set -e
+
+cidir=$(dirname "$0")
+testname="${0##*/}"
+sysname=$(uname)
+
+[ "$sysname"x == "Linux"x ] || exit 0
```

Comment:
I know it's traditional, but I don't think we need to use this form of syntax any more as bash is sane :) I'd remove the `x`'s and add a warning message that the test is being skipped.

Reply:
update